content/issues/2023-03-09-slo1-error-budget.md

```diff
@@ -0,0 +1,17 @@
+---
+title: 'SLO1 Error Budget issues'
+date: '2023-03-09T10:00:00+01:00'
+affected:
+  - API
+  - Workers
+  - Dashboard
+resolved: false
+section: issue
+severity: notice
+---
+
+We are having SLO1 error budget issues.
+
+These are mostly caused by performance issues and we are working on them and planning more improvements.
```

Comment:
Should we mention that we will prioritise fixing the SLO-related issues over working on other features?